catalogs/catalog-hadoop/src/main/java/org/apache/gravitino/catalog/hadoop/DefaultConfigurationProvider.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.gravitino.catalog.hadoop;
+
+import java.util.Map;
+import org.apache.hadoop.conf.Configuration;
+
+public class DefaultConfigurationProvider implements ConfigurationProvider {
+  @Override
+  public Configuration getConfiguration(Map<String, String> conf) {
+    Configuration configuration = new Configuration();
+    conf.forEach(configuration::set);
```

Comment:
Why do you pass all of the configurations into the hadoop conf? I think it's enough to pass in the configurations that start with `gravitino.bypass` in the default implementation.

Reply:
It seems that the current implementation is putting all configurations including keys without the prefix `gravitino.bypass`

Reply:
I think it's not appropriate, it might be a mistake before, can you confirm this?

Reply:
Yeah, I have changed the logic and only those with prefix `gravitino.bypass` can pass to the Hadoop configuration, it's indeed more reasonable.

Reply:
I found the line 137 of `HadoopCatalogOperations` still passes all the configuration into the Hadoop conf, this should be removed either.